release

```diff
@@ -8,6 +8,7 @@ do
       BRANCH="${1}"
       ;;
     -d|--deployment)
+      shift
```

Comment:
with one shift  `DEPLOYMENT = "-d"` instead of the value of the flag. There might be a more elegant way to do this but it works now when I do `release -b master -d production`